resources/default-settings/jukebox.default.yaml

```diff
@@ -87,6 +87,10 @@ playermpd:
     update_on_startup: true
     check_user_rights: true
   mpd_conf: ~/.config/mpd/mpd.conf
+  play_position_tracker:
```

Comment:
May just `resume` is a more intuitive name for users than the class name?

Reply:
Yeah, I thought the same right when seeing the diff of the PR. I guess I'm going to change the config option and possibly also the class name.

Reply:
Changed.